CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 1.1.0 (2021-02-03)
+## Product-wide changes
```

Comment:
Recommend calling this "Supported systems changes" or "Platform Support changes" to have similar language as our [public doc page](https://docs.microsoft.com/en-us/azure/iot-edge/support) on platform support.

Reply:
Fixed